packages/block-library/src/image/edit.native.js

```diff
@@ -91,6 +91,29 @@ export class ImageEdit extends Component {
 		this.accessibilityLabelCreator = this.accessibilityLabelCreator.bind(
 			this
 		);
+		this.setMappedAttributes = this.setMappedAttributes.bind( this );
+		this.onSizeChangeValue = this.onSizeChangeValue.bind( this );
+
+		this.options = {
+			url: {
+				label: __( 'Image Link URL' ),
+				placeholder: __( 'Add URL' ),
+				autoFocus: false,
+				autoFill: true,
+			},
+			openInNewTab: {
+				label: __( 'Open in new tab' ),
+			},
+			linkRel: {
+				label: __( 'Link Rel' ),
+				placeholder: __( 'None' ),
+			},
+		};
+
+		this.sizeOptions = map( this.props.imageSizes, ( { name, slug } ) => ( {
```

Comment:
I'm afraid that it can not be used as class property since it should be updated when the `this.props.imageSizes` is updated. Maybe you could use the state for it and `getDerivedStateFromProps` ?

Reply:
I think there is no need for that since `imageSizes` come from defaults settings and shouldn't change dynamically.